plugins/net_plugin/tests/auto_bp_peering_unittest.cpp

```diff
@@ -121,78 +131,52 @@ const eosio::chain::producer_authority_schedule test_schedule1{
    { { "proda"_n, {} }, { "prodb"_n, {} }, { "prodc"_n, {} }, { "prodd"_n, {} }, { "prode"_n, {} }, { "prodf"_n, {} },
      { "prodg"_n, {} }, { "prodh"_n, {} }, { "prodi"_n, {} }, { "prodj"_n, {} }, { "prodk"_n, {} }, { "prodl"_n, {} },
      { "prodm"_n, {} }, { "prodn"_n, {} }, { "prodo"_n, {} }, { "prodp"_n, {} }, { "prodq"_n, {} }, { "prodr"_n, {} },
-     { "prods"_n, {} }, { "prodt"_n, {} }, { "produ"_n, {} } }
+     { "prods"_n, {} }, /*{ "prodt"_n, {} },*/ { "produ"_n, {} } }
 };
 
 const eosio::chain::producer_authority_schedule test_schedule2{
    2,
    { { "proda"_n, {} }, { "prode"_n, {} }, { "prodi"_n, {} }, { "prodm"_n, {} }, { "prodp"_n, {} }, { "prods"_n, {} },
-     { "prodb"_n, {} }, { "prodf"_n, {} }, { "prodj"_n, {} }, { "prodn"_n, {} }, { "prodq"_n, {} }, { "prodt"_n, {} },
+     { "prodb"_n, {} }, { "prodf"_n, {} }, { "prodj"_n, {} }, { "prodn"_n, {} }, { "prodq"_n, {} }, /*{ "prodt"_n, {} },*/
      { "prodc"_n, {} }, { "prodg"_n, {} }, { "prodk"_n, {} }, { "prodo"_n, {} }, { "prodr"_n, {} }, { "produ"_n, {} },
      { "prodd"_n, {} }, { "prodh"_n, {} }, { "prodl"_n, {} } }
 };
 
-const eosio::chain::producer_authority_schedule reset_schedule1{ 1, {} };
-
-BOOST_AUTO_TEST_CASE(test_neighbor_finder) {
+const fc::flat_set<eosio::chain::account_name> producers_minus_prodkt{
+   "proda"_n, "prodb"_n, "prodc"_n, "prodd"_n, "prode"_n, "prodf"_n,
+   "prodg"_n, "prodh"_n, "prodi"_n, "prodj"_n, /*"prodk"_n,*/ "prodl"_n,
+   "prodm"_n, "prodn"_n, "prodo"_n, "prodp"_n, "prodq"_n, "prodr"_n,
+   "prods"_n, /*"prodt"_n,*/ "produ"_n };
```

Comment:
couple items commented out here and above. Maybe consider removing them; it's not clear if it's intentional or not

Reply:
I think it is intentional, the list is named `producers_minus_prodkt` and `prodk` and `prodt` are commented out for clarity.

Reply:
ah true for this new one the name does make it clearer. But the two above don't have such an indication though, and they weren't previously commented out; only newly commented out in this PR, that made those more sus to me

Reply:
lgtm